src/opencv.mk

```diff
@@ -21,6 +21,7 @@ endef
 define $(PKG)_BUILD
     # build
     cd '$(BUILD_DIR)' && '$(TARGET)-cmake' '$(SOURCE_DIR)' \
+      -DCMAKE_INSTALL_PREFIX='$(PREFIX)/$(TARGET)' \
```

Comment:
`'$(TARGET)-cmake'` sets `CMAKE_INSTALL_PREFIX` implicitly.

Reply:
Except it didn't. It was installing to mxe/tmp-opencv-i686-w64-mingw32.static/install and subsequently deleting every installed file.

Reply:
opencv's CMakeLists.txt has the following line:

```
set(CMAKE_INSTALL_PREFIX "${CMAKE_BINARY_DIR}/install" CACHE PATH "Installation Directory")
```

I think it overrides the value set by mxe-conf.cmake.

Probably we need to move the setting from mxe-conf.cmake to `src/cmake/conf/target-cmake.in`. Can you add TODO, please?

Reply:
Actually, I believe found the error. The file has two instances of that line; the patch currently in master patched both, mine was only patching the first instance. Let me see if correcting the patch fixes it.